scripts/linters/pylint_extensions_test.py

```diff
@@ -2862,3 +2862,50 @@ def test_single_space_after_keyword(self):
 
         with self.checker_test_object.assertNoMessages():
             temp_file.close()
+
+
+class InequalityWithNoneCheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        super(InequalityWithNoneCheckerTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.InequalityWithNoneChecker)
+        self.checker_test_object.setup_method()
+
+    def test_inequality_op_on_none_adds_message(self):
+        (if_node_one, if_node_two) = astroid.extract_node(
+            """
+            if x != None: #@
+                pass
+            if x != ( #@
+                None
+            ):
+                pass
+            """
+        )
+        compare_node_one = if_node_one.test
+        compare_node_two = if_node_two.test
+
+        not_equal_none_message_one = testutils.Message(
+            msg_id='inequality-with-none', node=compare_node_one)
+        not_equal_none_message_two = testutils.Message(
+            msg_id='inequality-with-none', node=compare_node_two)
+
+        with self.checker_test_object.assertAddsMessages(
+            not_equal_none_message_one,
+            not_equal_none_message_two
+        ):
+            self.checker_test_object.checker.visit_compare(compare_node_one)
+            self.checker_test_object.checker.visit_compare(compare_node_two)
+
+    def test_usage_of_is_not_on_none_does_not_add_message(self):
+        (if_node) = astroid.extract_node(
```

Comment:
Nit: Do we need parenthesis here? (If not let's remove it! :) )

Reply:
Done!